### PR TITLE
Use ModDataChecker feature to check content of archive

### PR DESCRIPTION
**Pull request by Holt59**
**WIP** because I want to change a bit the UI.

The new interface:

- No `ModDataChecker` - Clicking "Ok" will not warn the user:

image

- Invalid data:

image

image

- Valid data:

image

### CODE REVIEW

File: src/installermanual.cpp [hunk 1]
```diff
@@ -50,7 +53,7 @@ QString InstallerManual::name() const
 
 QString InstallerManual::author() const
 {
-  return "Tannin";
+  return "Tannin, Holt59";
```
Comment (Al12rs) [inline src/installermanual.cpp:56]: You know this means future maintainer XD

Comment (Holt59) [reply src/installermanual.cpp:56]: I even forgot I put that here. 🙃

**Comment by Al12rs** [comment]
Instead of cannot check the contents I would write something else. We get people that see that dialog and have no idea what to do so maybe we should say something like "Please define to install the files of this mod" or something.

**Comment by Holt59** [comment]
> Instead of cannot check the contents I would write something else. We get people that see that dialog and have no idea what to do so maybe we should say something like "Please define to install the files of this mod" or something.
> "Cannot check the contents" sounds like an error and users might think that is an error dialog or something. I would use an imperative to tell them they need to tell Mo2 how to install the mod.

I don't understand your "Please define [...]."

**Comment by Al12rs** [comment]
I was missing a "how"
So "Please define how to install the files of this mod" was my proposition but there is probably better.

**Pull request merged**